ftd_django/template_backend.py

```diff
@@ -0,0 +1,71 @@
+from django.template.backends.base import BaseEngine
+from django.template.backends.utils import csrf_input_lazy, csrf_token_lazy
+import ftd_django.template_library
+import errno
+import os
+
+from django.conf import settings
+
+
+class FTDTemplateBackend(BaseEngine):
+
+    # Name of the subdirectory containing the templates for this engine
+    # inside an installed application.
+    app_dirname = "templates"
+
+    def __init__(self, params):
+        params = params.copy()
+        options = params.pop("OPTIONS").copy()
+        options.setdefault("autoescape", True)
+        options.setdefault("debug", settings.DEBUG)
+        options.setdefault("file_charset", "utf-8")
+        # libraries = options.get("libraries", {})
+        # options["libraries"] = self.get_templatetag_libraries(libraries)
+        super().__init__(params)
+        # self.engine = Engine(self.dirs, self.app_dirs, **options)
+
+    def from_string(self, template_code):
+        path = "templates/" + template_code
+        if not os.path.exists(path):
+            raise FileNotFoundError(errno.ENOENT, os.strerror(errno.ENOENT), path)
+        return Template(path)
```

Comment:
Raise NotImplemented exception from here. We do not support `from_string()` yet.